internal/admin/authorizedapps/save.go

```diff
@@ -27,16 +27,16 @@ import (
 	"github.com/google/exposure-notifications-server/internal/serverenv"
 )
 
-type saveController struct {
+type SaveController struct {
```

Comment:
Then these types don't need to be exported (unless the linter complains I guess)

Reply:
Done